docs/changelog.rst

```diff
@@ -1,5 +1,29 @@
 Changelog
 ================================
+CBC SDK 1.3.4 - Released [TBD]
+------------------------------
+
+New Features:
+
+* Recommendations API - suggested reputation overrides for policy configuration.
+
+Updates:
+
+* Improved string representation of objects through ``__str__()`` mechanism.
+
+Bug Fixes:
+
+* Ensure proper ``TimeoutError`` is raised in several places where the wrong exception was being raised.
+* Fix to allowed categories when performing alert queries.
+
+Documentation Fixes:
```

Comment:
Instead of Documentation Fixes can we call it Documentation Changes given this includes both fixes and new content